ognjen/pay-pal/src/components/ItemCard.jsx

```diff
@@ -0,0 +1,23 @@
+import Button from 'react-bootstrap/Button';
+
+export default function ItemCard ({title}) {
+    const cardImg = {
+        Individuals: 'https://www.paypalobjects.com/digitalassets/c/website/marketing/emea/gb/en/home/woman_side_icon.jpeg',
+        Buisnisses: 'https://www.paypalobjects.com/digitalassets/c/website/marketing/emea/gb/en/home/pos_machine_icon.jpeg',
+        Partners: 'https://www.paypalobjects.com/digitalassets/c/website/marketing/emea/gb/en/home/shopping_bag_icon.jpeg'
+    }
+    const description = {
+        Individuals: 'Find out why we have more than 200M active accounts worldwide.',
+        Buisnisses: 'Join more than 7 million businesses around the world offering PayPal.',
+        Partners: 'Work with PayPal to offer your customers world class payment solutions.'
+    }
```

Comment:
I'd suggest inverting these so we avoid duplications.
```suggestion
    const cardData = {
        Individuals: {
           cardImg: "https://www.paypalobjects.com/digitalassets/c/website/marketing/emea/gb/en/home/woman_side_icon.jpeg",
           description: "Find out why we have more than 200M active accounts worldwide."
       },
       Buisnisses: {
           cardImg: "https://www.paypalobjects.com/digitalassets/c/website/marketing/emea/gb/en/home/pos_machine_icon.jpeg",
           description: "Join more than 7 million businesses around the world offering PayPal."
       },
        Partners: {
           cardImg: "https://www.paypalobjects.com/digitalassets/c/website/marketing/emea/gb/en/home/shopping_bag_icon.jpeg",
           description: "Work with PayPal to offer your customers world class payment solutions."
       }
    }
```